app/controllers/concerns/rate_limit_concern.rb

```diff
@@ -64,8 +64,8 @@ def idv_attempter_rate_limited?(rate_limit_type)
   end
 
   def pii_ssn
-    return unless defined?(flow_session) && user_session
-    pii_from_doc_ssn = flow_session[:pii_from_doc]&.[](:ssn)
+    return unless defined?(flow_session) && defined?(idv_session) && user_session
+    pii_from_doc_ssn = idv_session&.ssn || flow_session[:pii_from_doc]&.[](:ssn)
     return pii_from_doc_ssn if pii_from_doc_ssn
     flow_session[:pii_from_user]&.[](:ssn)
```

Comment:
Initially I thought we needed to make an update to line 70 to include an or statement but whether Remote (pii_from_doc) or IPP (pii_from_user) - if idv_session.ssn was present, line 68 would return it so it would not make it past.  Nothing to change - just writing my thoughts.